packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -460,16 +391,16 @@ void _applyCameraUpdate(gmaps.GMap map, CameraUpdate update) {
           // print('Error computing new focus LatLng. JS Error: ' + e.toString());
         }
       }
-      map.zoom = map.zoom + newZoomDelta;
+      map.zoom = map.zoom! + newZoomDelta;
```

Comment:
Why are all these `zoom!`'s safe?

Reply:
I just can't imagine how an user can get to this point of the code having null `zoom` in their Maps :)

Reply:
Replaced by `(map.zoom ?? 0)` and made 0 the default value for zoom (according to the JS api, when zoom is not initialized)

Reply:
On JS side you can do really strange things :-/

```
> map.setZoom(null)
undefined
> map.getZoom()
null
```